Jenkinsfile-deploy

```diff
@@ -0,0 +1,20 @@
+library 'whatsout'
+
+node('deploy-kubernetes'){
```

Comment:
used for internal deploy with jenkins

Reply:
It feels weird to have our specific in-house deployment code in the open source repo where no one else will be able to use it anyway. I guess it's not harmful, but it increases complexity for everyone else without providing value.

Same for config.json and the kube-folder.

Reply:
The kube folder is universal. Its only the config.json that is not. The complexity increased by moving it to another repository is even higher.

Reply:
But i agree, its not fun to have it here.